### ISSUE TITLE
dvpackager: CLI/GUI won't package file (audio rate change)

### ISSUE THREAD

**Issue by libbyhopfauf** [mipops/dvrescue#759]
### Overview
I thought this might be a helpful test example, but I'm not sure what's going on. Using the latest version of dvrescue, both the CLI and GUI keep failing to package this file. There is a change in audio rate (from 0 to 4800) right at the beginning the beginning. Video signal is present throughout and does not change when the audio rate does. All of the related files and logs are available [here](https://drive.google.com/drive/folders/1xhloUAtEqKuBqQxReHR_xq58HCpgDI_I?usp=sharing) for review. I've tested this with versions 2023-11-11 through 2023-11-27-1. This might be similar to issue #726.

### Testing 
**GUI** 

1. When separating by audio rate, the GUI will only output part1 (a few frames of silence) and then fails when trying to package part2. 

<img width="1512" alt="12964_dvpackager-GUI_fail" src="https://github.com/mipops/dvrescue/assets/13877631/3e6c027a-8e1a-4f1a-8e7e-2c7fb078bc5b">

2. As expected, when no packaging options are selected, the GUI outputs a single file with the audio rate set to 0 (which is what it is for the first few frames before changing to 4800).

**CLI** - I tried a variety of output options for the CLI, but can still not get dvpackager to separate the dv file into two separate files based on audio rate or to force 4800 to be the audio rate. The CLI will do one of two things depending on the specifications for how to handle audio rate:

1. If I specify that dvpackager should force the audio to 4800 (`-4`), it will only output the second part, but keep encoding the audio as 0. Based on the messages in the Terminal, apparently ffmpeg and dvrescue don't agree on which audio to output for either segment and it opts to go with ffmpeg (which in this case is incorrect). I keep getting the following error:

```
### Packaging started: /Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/SMA/12964_part1.mkv
ERROR: ffmpeg and dvrescue disagree about the audio characterstics at offset 0 at 00:00:00.000000.
We shall use ffmpeg's analysis on this. (ff:2-48000 != dvr:0)
Consider updating to the latest version of dvrescue and this error recurs,
then consider mentioning this at https://github.com/mipops/dvrescue/issues.
FYI: Padding a little silence near 00:00:00.333666 for 12964_part1.mkv since a source portion ends with frames missing audio (via ffmpeg).
ERROR: ffmpeg and dvrescue disagree about the audio characterstics at offset 1200000 at 00:00:00.333666.
We shall use ffmpeg's analysis on this. (ff:0 != dvr:2-48000)
Consider updating to the latest version of dvrescue and this error recurs,
then consider mentioning this at https://github.com/mipops/dvrescue/issues.
FYI: Padding a little silence near 00:41:11.435633 for 12964_part1.mkv since a source portion ends with frames missing audio (via ffmpeg).
```

2. If I rearrange the options and try to force the audio rate, I get a single file of silence and the same error message as described above.

3. If I try to separate the file by audio rate (using `-3`), I get the following error:

```
ERROR: 12964_part2.mkv is expected but missing. Please retry with the -v option and report to https://github.com/mipops/dvrescue/issues.
ERROR: 12964.dv contained 74069 DV frames, but the outputs contain 10 frames of DV and 0 were skipped
```

4. Adding the `-v` option provides the following:
```
mipops-macbook@mipops-macbooks-MacBook-Pro ~ % dvpackager -o /Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/dvpackager-outputs/CLI/separate-audio-rate-v -e mkv -3 -v /Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/original-files/12964.dv

#CONDENSE_OPTS is set to 
#Using /opt/homebrew/opt/ffmpegdecklink/bin/ffmpeg-dl for ffmpeg operations.
Packaging 12964.dv, making dvrescue outputs
File '/var/folders/ck/8qhr5yvd523977jlkkgpyx3w0000gn/T/dvpackager.XXXXXX.hc1N07do' already exists. Overwrite? [y/N] y
#No xmlstarlet error000-00-00 00:00:00;00
#Trying to move the temporary xml from /var/folders/ck/8qhr5yvd523977jlkkgpyx3w0000gn/T/dvpackager.XXXXXX.hc1N07do to /Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/dvpackager-outputs/CLI/separate-audio-rate-v/12964.dv.dvrescue.xml
#DV_FRAME_COUNT=(74069)
#SOURCE_FORMAT=(DV)
#Trying to present a table of ranges

# Segmentation options. Split on: Recording_Start_Marker = 0, Recording_Timestamp_Jump = 0, Timecode_Jump = 0, Audio_characteristics_change = 1, Aspect_Ratio_change = 1
#         St='Flagged Start of a recording', ncTC='non-continuous timecode value', ncR='non-continuous recording timestamp value'
  # | PTS Range                         | Duration | Frame Range         | Byte Range                | Timecode    | Recording Timestamp    | Size      | Frame Rate | DAR   | ChSub | Audio     | St | ncTC | ncR |
  1 | 00:00:00.000000 - 00:00:00.333666 |    0.334 |        0 -        9 |           0 -     1200000 | 00:00:01;17 | 2003-12-24 12:43:41    |   720x480 | 30000/1001 |   4/3 | 4:1:1 | 0ch     0 |    |      |     |
  2 | 00:00:00.333666 - 00:41:11.435633 | 2471.102 |       10 -    74068 |     1200000 -  8888280000 | 00:00:01;17 | 2003-12-24 12:43:41    |   720x480 | 30000/1001 |   4/3 | 4:1:1 | 2ch 48000 |    |      |     |

The results will be written to /Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/dvpackager-outputs/CLI/separate-audio-rate-v
### Packaging started: /Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/dvpackager-outputs/CLI/separate-audio-rate-v/12964_part1.mkv
#Reading from DV stream input at byte range 0-1200000
#The audio range sublist (from frame 0 to 9) is
#00:00:00.000000|00:00:00.333666|0|9|00:00:01;17|0|2003-12-24 12:43:41|720x480|30000/1001|4:1:1|4/3|0|0||||0|1200000|1080000
#Prepping a mapping for 0 channel input ().
#Writing a chapter track at /var/folders/ck/8qhr5yvd523977jlkkgpyx3w0000gn/T/dvpackager.XXXXXX.sq034mO1.ffmetadata.
#Found 1 point(s) for chapter marks.
#Skipping the process to embed chapters since there would only be one.
#Trying to run: /opt/homebrew/opt/ffmpegdecklink/bin/ffmpeg-dl  -y -nostdin -hide_banner -i subfile,,start,0,end,1200000,,:/Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/original-files/12964.dv     -t 0.333666667 -c:a pcm_s16le -c:v:0 copy -f matroska -map 0:v:0 -map 0:a? -metadata timecode=00:00:01;17 -metadata DATE_RECORDED=2003-12-24 12:43:41 -aspect 4/3 /Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/dvpackager-outputs/CLI/separate-audio-rate-v/12964_part1.mkv
[dv @ 0x12ff04080] Detected timecode is invalid
[dv @ 0x12ff04080] Estimating duration from bitrate, this may be inaccurate
Input #0, dv, from 'subfile,,start,0,end,1200000,,:/Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/original-files/12964.dv':
  Duration: 00:00:00.33, start: 0.000000, bitrate: 28771 kb/s
  Stream #0:0: Video: dvvideo, yuv411p, 720x480 [SAR 8:9 DAR 4:3], 25000 kb/s, 29.97 fps, 29.97 tbr, 29.97 tbn
  Stream #0:1: Audio: pcm_s16le, 48000 Hz, stereo, s16, 1536 kb/s
Overriding aspect ratio with stream copy may produce invalid files
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
  Stream #0:1 -> #0:1 (pcm_s16le (native) -> pcm_s16le (native))
Output #0, matroska, to '/Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/dvpackager-outputs/CLI/separate-audio-rate-v/12964_part1.mkv':
  Metadata:
    timecode        : 00:00:01;17
    DATE_RECORDED   : 2003-12-24 12:43:41
    encoder         : Lavf59.27.100
  Stream #0:0: Video: dvvideo (dvsd / 0x64737664), yuv411p, 720x480 [SAR 8:9 DAR 4:3], q=2-31, 25000 kb/s, 29.97 fps, 29.97 tbr, 1k tbn
  Stream #0:1: Audio: pcm_s16le ([1][0][0][0] / 0x0001), 48000 Hz, stereo, s16, 1536 kb/s
    Metadata:
      encoder         : Lavc59.37.100 pcm_s16le
frame=    1 fps=0.0 q=-1.0 size=       1kB time=00:00:00.03 bitrate= 189.3kbits/frame=   10 fps=0.0 q=-1.0 Lsize=    1236kB time=00:00:00.33 bitrate=30307.4kbits/s speed=78.3x    
video:1172kB audio:63kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.102370%
#AUD_STREAM_COUNT=(2)
#VID_DUR=(333.000000), AUD_DIR=(333.000000), DUR_DIFF=(0.000000)
#DUR_DIFF is within an acceptable range.
### Done:seg_input=1|seg_output=1|output_frames=10|input_frames=10|audio_fill=0|output_file=/Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/dvpackager-outputs/CLI/separate-audio-rate-v/12964_part1.mkv
### Packaging finished: /Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/dvpackager-outputs/CLI/separate-audio-rate-v/12964_part1.mkv
### Packaging started: /Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/dvpackager-outputs/CLI/separate-audio-rate-v/12964_part2.mkv
#Reading from DV stream input at byte range 1200000-8888280000
#The audio range sublist (from frame 10 to 74068) is
#00:00:00.333666|00:41:11.435633|10|74068|00:00:01;17|1200000|2003-12-24 12:43:41|720x480|30000/1001|4:1:1|4/3|48000|2||||0|8888280000|8888160000
#Prepping a mapping for 2 channel input ().
#Writing a chapter track at /var/folders/ck/8qhr5yvd523977jlkkgpyx3w0000gn/T/dvpackager.XXXXXX.nuaVGq54.ffmetadata.
#Found 41 point(s) for chapter marks.
#Chapter: 00:00:00.333666-00:00:45.011633 (0-4467797): 00:00:01;17 - 2003-12-24 12:43:41
#Chapter: 00:00:45.011633-00:00:50.350300 (4467797-5001664): 00:00:44;27 - 2003-12-24 12:44:31
#Chapter: 00:00:50.350300-00:00:57.490766 (5001664-5715710): 00:00:50;07 - 2003-12-24 12:44:49
#Chapter: 00:00:57.490766-00:01:15.942533 (5715710-7560887): 00:00:57;11 - 2003-12-24 12:44:59
#Chapter: 00:01:15.942533-00:01:27.353933 (7560887-8702027): 00:01:15;26 - 2003-12-24 12:45:21
#Chapter: 00:01:27.353933-00:01:38.231466 (8702027-9789780): 00:01:27;08 - 2003-12-24 12:45:39
#Chapter: 00:01:38.231466-00:01:44.270833 (9789780-10393717): 00:01:38;04 - 2003-12-24 12:45:56
#Chapter: 00:01:44.270833-00:01:56.783333 (10393717-11644967): 00:01:44;05 - 2003-12-24 12:46:09
#Chapter: 00:01:56.783333-00:02:05.458666 (11644967-12512500): 00:01:56;20 - 2003-12-24 12:46:31
#Chapter: 00:02:05.458666-00:02:18.805333 (12512500-13847167): 00:02:05;12 - 2003-12-24 12:46:45
#Chapter: 00:02:18.805333-00:02:32.385566 (13847167-15205190): 00:02:18;22 - 2003-12-24 12:48:16
#Chapter: 00:02:32.385566-00:02:46.166000 (15205190-16583234): 00:02:32;09 - 2003-12-24 12:48:58
#Chapter: 00:02:46.166000-00:03:19.532666 (16583234-19919900): 00:02:46;02 - 2003-12-24 12:49:16
#Chapter: 00:03:19.532666-00:03:27.774233 (19919900-20744057): 00:03:19;14 - 2003-12-24 12:49:59
#Chapter: 00:03:27.774233-00:03:48.695133 (20744057-22836147): 00:03:27;21 - 2003-12-24 12:50:35
#Chapter: 00:03:48.695133-00:04:08.715133 (22836147-24838147): 00:03:48;18 - 2003-12-24 12:51:20
#Chapter: 00:04:08.715133-00:04:16.356100 (24838147-25602244): 00:04:08;20 - 2003-12-24 12:53:45
#Chapter: 00:04:16.356100-00:04:25.198266 (25602244-26486460): 00:04:16;09 - 2003-12-24 12:53:57
#Chapter: 00:04:25.198266-00:04:38.311366 (26486460-27797770): 00:04:25;04 - 2003-12-24 12:54:12
#Chapter: 00:04:38.311366-00:04:39.912966 (27797770-27957930): 00:04:38;07 - 2003-12-24 12:54:29
#Chapter: 00:04:39.912966-00:04:55.161533 (27957930-29482787): 00:04:39;25 - 2003-12-24 12:54:46
#Chapter: 00:04:55.161533-00:04:59.132166 (29482787-29879850): 00:04:55;02 - 2003-12-24 12:56:35
#Chapter: 00:04:59.132166-00:05:07.507200 (29879850-30717354): 00:04:59;01 - 2003-12-24 12:56:58
#Chapter: 00:05:07.507200-00:06:03.096066 (30717354-36276240): 00:05:07;14 - 2003-12-24 12:57:27
#Chapter: 00:06:03.096066-00:06:07.900866 (36276240-36756720): 00:06:03;02 - 2003-12-24 12:58:48
#Chapter: 00:06:07.900866-00:06:21.114066 (36756720-38078040): 00:06:07;26 - 2003-12-24 12:59:03
#Chapter: 00:06:21.114066-00:06:28.187800 (38078040-38785414): 00:06:21;02 - 2003-12-24 12:59:35
#Chapter: 00:06:28.187800-00:06:33.626566 (38785414-39329290): 00:06:28;04 - 2003-12-24 12:59:47
#Chapter: 00:06:33.626566-00:06:52.345266 (39329290-41201160): 00:06:33;17 - 2003-12-24 13:00:04
#Chapter: 00:06:52.345266-00:07:07.560466 (41201160-42722680): 00:06:52;08 - 2003-12-24 13:00:46
#Chapter: 00:07:07.560466-00:07:37.023233 (42722680-45668957): 00:07:07;16 - 2003-12-24 13:01:03
#Chapter: 00:07:37.023233-00:08:01.013866 (45668957-48068020): 00:07:36;29 - 2003-12-24 13:01:35
#Chapter: 00:08:01.013866-00:08:37.016499 (48068020-51668283): 00:08:01;00 - 2003-12-24 13:02:24
#Chapter: 00:08:37.016499-00:08:54.333800 (51668283-53400014): 00:08:36;29 - 2003-12-24 13:03:12
#Chapter: 00:08:54.333800-00:09:07.580366 (53400014-54724670): 00:08:54;08 - 2003-12-24 13:21:38
#Chapter: 00:09:07.580366-00:09:18.891666 (54724670-55855800): 00:09:07;17 - 2003-12-24 13:22:08
#Chapter: 00:09:18.891666-00:09:53.726466 (55855800-59339280): 00:09:18;26 - 2003-12-24 13:22:40
#Chapter: 00:09:53.726466-00:10:01.767833 (59339280-60143417): 00:09:53;20 - 2003-12-24 13:23:25
#Chapter: 00:10:01.767833-00:10:29.128500 (60143417-62879484): 00:10:01;21 - 2003-12-24 13:24:37
#Chapter: 00:10:29.128500-00:30:19.984833 (62879484-181965117): 02:57:56;16 - 2000-00-00 00:00:00;00
#Chapter: 00:30:19.984833-00:41:11.435633 (181965117-247110197): 03:17:47;11 - 2000-00-00 00:00:00;00
#Trying to run: /opt/homebrew/opt/ffmpegdecklink/bin/ffmpeg-dl  -y -nostdin -hide_banner -i subfile,,start,1200000,end,8888280000,,:/Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/original-files/12964.dv   -i /var/folders/ck/8qhr5yvd523977jlkkgpyx3w0000gn/T/dvpackager.XXXXXX.nuaVGq54.ffmetadata -map_metadata 1 -filter_complex [0:a:0]aresample=async=1:min_hard_comp=0.01[aud] -t 2471.101966667 -c:a pcm_s16le -c:v:0 copy -f matroska -map 0:v:0 -map [aud] -metadata timecode=00:00:01;17 -metadata DATE_RECORDED=2003-12-24 12:43:41 -aspect 4/3 /Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/dvpackager-outputs/CLI/separate-audio-rate-v/12964_part2.mkv
[dv @ 0x13ff04570] Detected timecode is invalid
[dv @ 0x13ff04570] Estimating duration from bitrate, this may be inaccurate
Input #0, dv, from 'subfile,,start,1200000,end,8888280000,,:/Users/mipops-macbook/Desktop/2023-11-06_vrecord-test/12964/original-files/12964.dv':
  Duration: 00:41:11.10, start: 0.000000, bitrate: 28771 kb/s
  Stream #0:0: Video: dvvideo, yuv411p, 720x480 [SAR 8:9 DAR 4:3], 25000 kb/s, 29.97 fps, 29.97 tbr, 29.97 tbn
Input #1, ffmetadata, from '/var/folders/ck/8qhr5yvd523977jlkkgpyx3w0000gn/T/dvpackager.XXXXXX.nuaVGq54.ffmetadata':
  Duration: 00:41:11.10, start: 0.000000, bitrate: 0 kb/s
  Chapters:
    Chapter #1:0: start 0.000000, end 44.677970
      Metadata:
        title           : 00:00:01;17 - 2003-12-24 12:43:41
    Chapter #1:1: start 44.677970, end 50.016640
      Metadata:
        title           : 00:00:44;27 - 2003-12-24 12:44:31
    Chapter #1:2: start 50.016640, end 57.157100
      Metadata:
        title           : 00:00:50;07 - 2003-12-24 12:44:49
    Chapter #1:3: start 57.157100, end 75.608870
      Metadata:
        title           : 00:00:57;11 - 2003-12-24 12:44:59
    Chapter #1:4: start 75.608870, end 87.020270
      Metadata:
        title           : 00:01:15;26 - 2003-12-24 12:45:21
    Chapter #1:5: start 87.020270, end 97.897800
      Metadata:
        title           : 00:01:27;08 - 2003-12-24 12:45:39
    Chapter #1:6: start 97.897800, end 103.937170
      Metadata:
        title           : 00:01:38;04 - 2003-12-24 12:45:56
    Chapter #1:7: start 103.937170, end 116.449670
      Metadata:
        title           : 00:01:44;05 - 2003-12-24 12:46:09
    Chapter #1:8: start 116.449670, end 125.125000
      Metadata:
        title           : 00:01:56;20 - 2003-12-24 12:46:31
    Chapter #1:9: start 125.125000, end 138.471670
      Metadata:
        title           : 00:02:05;12 - 2003-12-24 12:46:45
    Chapter #1:10: start 138.471670, end 152.051900
      Metadata:
        title           : 00:02:18;22 - 2003-12-24 12:48:16
    Chapter #1:11: start 152.051900, end 165.832340
      Metadata:
        title           : 00:02:32;09 - 2003-12-24 12:48:58
    Chapter #1:12: start 165.832340, end 199.199000
      Metadata:
        title           : 00:02:46;02 - 2003-12-24 12:49:16
    Chapter #1:13: start 199.199000, end 207.440570
      Metadata:
        title           : 00:03:19;14 - 2003-12-24 12:49:59
    Chapter #1:14: start 207.440570, end 228.361470
      Metadata:
        title           : 00:03:27;21 - 2003-12-24 12:50:35
    Chapter #1:15: start 228.361470, end 248.381470
      Metadata:
        title           : 00:03:48;18 - 2003-12-24 12:51:20
    Chapter #1:16: start 248.381470, end 256.022440
      Metadata:
        title           : 00:04:08;20 - 2003-12-24 12:53:45
    Chapter #1:17: start 256.022440, end 264.864600
      Metadata:
        title           : 00:04:16;09 - 2003-12-24 12:53:57
    Chapter #1:18: start 264.864600, end 277.977700
      Metadata:
        title           : 00:04:25;04 - 2003-12-24 12:54:12
    Chapter #1:19: start 277.977700, end 279.579300
      Metadata:
        title           : 00:04:38;07 - 2003-12-24 12:54:29
    Chapter #1:20: start 279.579300, end 294.827870
      Metadata:
        title           : 00:04:39;25 - 2003-12-24 12:54:46
    Chapter #1:21: start 294.827870, end 298.798500
      Metadata:
        title           : 00:04:55;02 - 2003-12-24 12:56:35
    Chapter #1:22: start 298.798500, end 307.173540
      Metadata:
        title           : 00:04:59;01 - 2003-12-24 12:56:58
    Chapter #1:23: start 307.173540, end 362.762400
      Metadata:
        title           : 00:05:07;14 - 2003-12-24 12:57:27
    Chapter #1:24: start 362.762400, end 367.567200
      Metadata:
        title           : 00:06:03;02 - 2003-12-24 12:58:48
    Chapter #1:25: start 367.567200, end 380.780400
      Metadata:
        title           : 00:06:07;26 - 2003-12-24 12:59:03
    Chapter #1:26: start 380.780400, end 387.854140
      Metadata:
        title           : 00:06:21;02 - 2003-12-24 12:59:35
    Chapter #1:27: start 387.854140, end 393.292900
      Metadata:
        title           : 00:06:28;04 - 2003-12-24 12:59:47
    Chapter #1:28: start 393.292900, end 412.011600
      Metadata:
        title           : 00:06:33;17 - 2003-12-24 13:00:04
    Chapter #1:29: start 412.011600, end 427.226800
      Metadata:
        title           : 00:06:52;08 - 2003-12-24 13:00:46
    Chapter #1:30: start 427.226800, end 456.689570
      Metadata:
        title           : 00:07:07;16 - 2003-12-24 13:01:03
    Chapter #1:31: start 456.689570, end 480.680200
      Metadata:
        title           : 00:07:36;29 - 2003-12-24 13:01:35
    Chapter #1:32: start 480.680200, end 516.682830
      Metadata:
        title           : 00:08:01;00 - 2003-12-24 13:02:24
    Chapter #1:33: start 516.682830, end 534.000140
      Metadata:
        title           : 00:08:36;29 - 2003-12-24 13:03:12
    Chapter #1:34: start 534.000140, end 547.246700
      Metadata:
        title           : 00:08:54;08 - 2003-12-24 13:21:38
    Chapter #1:35: start 547.246700, end 558.558000
      Metadata:
        title           : 00:09:07;17 - 2003-12-24 13:22:08
    Chapter #1:36: start 558.558000, end 593.392800
      Metadata:
        title           : 00:09:18;26 - 2003-12-24 13:22:40
    Chapter #1:37: start 593.392800, end 601.434170
      Metadata:
        title           : 00:09:53;20 - 2003-12-24 13:23:25
    Chapter #1:38: start 601.434170, end 628.794840
      Metadata:
        title           : 00:10:01;21 - 2003-12-24 13:24:37
    Chapter #1:39: start 628.794840, end 1819.651170
      Metadata:
        title           : 02:57:56;16 - 2000-00-00 00:00:00;00
    Chapter #1:40: start 1819.651170, end 2471.101970
      Metadata:
        title           : 03:17:47;11 - 2000-00-00 00:00:00;00
Stream specifier ':a:0' in filtergraph description [0:a:0]aresample=async=1:min_hard_comp=0.01[aud] matches no streams.
ERROR: 12964_part2.mkv is expected but missing. Please retry with the -v option and report to https://github.com/mipops/dvrescue/issues.
ERROR: 12964.dv contained 74069 DV frames, but the outputs contain 10 frames of DV and 0 were skipped
mipops-macbook@mipops-macbooks-MacBook-Pro ~ % 
```

 
### Additional notes for dvpackager behavior

- CLI: dvpackager -h says that the default wrapper is mkv, but if no extension is noted, it defaults to mov.
- GUI: the display icons are not always accurate (see issue #758).

<img width="1512" alt="dvpackager-GUI_icons-display-incorrect-2" src="https://github.com/mipops/dvrescue/assets/13877631/d9f18397-5b51-436d-8ba2-480ecaedcb8c">

**Comment by dericed** [comment]
Hi @libbyhopfauf please retest with the next daily build.

**Comment by libbyhopfauf** [comment]
> Hi @libbyhopfauf please retest with the next daily build.

Will do! Which build should I be testing with?

**Comment by dericed** [comment]
@libbyhopfauf the latest from https://old.mediaarea.net/download/snapshots/binary/dvrescue/ https://old.mediaarea.net/download/snapshots/binary/dvrescue-gui/

**Comment by dericed** [comment]
Hi @libbyhopfauf, I think I addressed this in https://github.com/mipops/dvrescue/commit/ce9cfdf3e79be09537802d5fd3c35ec589c4136e. Can you retest with the latest build?

**Comment by libbyhopfauf** [comment]
> Hi @libbyhopfauf, I think I addressed this in [ce9cfdf](https://github.com/mipops/dvrescue/commit/ce9cfdf3e79be09537802d5fd3c35ec589c4136e). Can you retest with the latest build?

In the GUI, the display said that the second part failed, but it was actually encoded properly. So it seems like the underlying problem is fixed and now the GUI just has the same issue as #758.
